### PR TITLE
Update Django versions

### PR DESCRIPTION
**Pull request by DmytroLitvinov**
Hello @jsocol,

Could you please review it and merge it?

### CODE REVIEW

**Comment by jsocol** [comment]
Hi there. I haven’t been a maintainer of this library for a number of years. Someone from Mozilla will need to help you out.

**Comment by DmytroLitvinov** [comment]
Unforunally, I don't know who is working under Mozilla and could help in this question.
Could you please tag people who could help with merging PR?

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+dist: xenial   # required for Python >= 3.7
```
Comment (g-k) [inline .travis.yml]: pypy and pypy3 aren't available on xenial, so I think we need to specify an older version in the build matrix

**Comment by pirate** [comment]
Would be nice to move from the old `MIDDLEWARE_CLASSES` to the new `MIDDLEWARE` system as well, the conversion is pretty easy:

```python
class SomeMiddleware:
    def process_request(request):
        # stuff before request here
        ...
        return request
    def process_response(request, response):
        # stuff after response here
        ...
        return response
```
To:
```python
def SomeMiddleware(get_response):
    def middleware(request):
        # stuff before request here
        ...

        response = get_response(request)

        # stuff after response here
        ...

        return response
```

I can submit a PR if you want once this is merged.  Django provides an upgrade path with backwards compatibility if you want to keep supporting the old `MIDDLEWARE_CLASSES` system and the newer version in parallel: https://docs.djangoproject.com/en/2.2/topics/http/middleware/#upgrading-pre-django-1-10-style-middleware

**Comment by DmytroLitvinov** [comment]
I am going to fix CI tests today/tomorrow.

**Pull request closed without merging**